apps.yml

```diff
@@ -1726,19 +1704,6 @@ apps:
     name: Mozilla People Directory
     op: auth0
     url: https://people.mozilla.org
-- application:
-    AAL: LOW
-    authorized_groups:
-    - everyone
-    authorized_users: []
-    client_id: o2e391VjmnPk0115UedNTmRL8x2nySOa
-    display: true
-    logo: people.png
-    name: Phonebook
-    op: auth0
-    url: https://people.mozilla.org
```

Comment:
This particular removal references people.mozilla.org is this removal correct?

Reply:
This is kinda correct. As Florian says above, that client ID is duplicated, because it is used also in [here](https://github.com/mozilla-iam/sso-dashboard-configuration/blob/master/apps.yml#L1743-L1753).